src/instance-id/instance-id-request.ts

```diff
@@ -94,26 +93,17 @@ export class FirebaseInstanceIdRequestHandler {
         return response;
       })
       .catch((response) => {
-        let error;
-        if (typeof response === 'object' && 'error' in response) {
-          error = response.error;
-        } else {
-          error = response;
-        }
-
+        const error = (typeof response === 'object' && 'error' in response)
+          ? response.error : response;
```

Comment:
Better for readability to have the ? on the previous line

Reply:
Done